agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/dockerbuild.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-docker build --no-cache -t agogosml -f agogosml/Dockerfile.agogosml agogosml/
-docker build --no-cache -t input_reader -f input_reader/Dockerfile.input_reader input_reader/
-docker build --no-cache -t output_writer -f output_writer/Dockerfile.output_writer output_writer/
-docker build --no-cache -t sample_app -f {{cookiecutter.PROJECT_NAME_SLUG}}/Dockerfile.{{cookiecutter.PROJECT_NAME_SLUG}} {{cookiecutter.PROJECT_NAME_SLUG}}/
+docker build -t agogosml -f agogosml/Dockerfile.agogosml agogosml/
+docker build -t input_reader -f input_reader/Dockerfile.input_reader input_reader/
+docker build -t output_writer -f output_writer/Dockerfile.output_writer output_writer/
+docker build -t sample_app -f <YOUR_APP_NAME>/Dockerfile.<YOUR_APP_NAME> <YOUR_APP_NAME>/
```

Comment:
Why was this changed? cookiecutter was injecting the name of the project directly into the dockerbuild.sh file.

Reply:
ohhh silly me, thanks!